src/System.Runtime.Serialization.Formatters/tests/EqualityExtensions.cs

```diff
@@ -118,7 +118,7 @@ public static void CheckSequenceEquals(this IEnumerable @this, IEnumerable other
             try
             {
                 eA = (@this as IEnumerable).GetEnumerator();
-                eB = (@this as IEnumerable).GetEnumerator();
+                eB = (other as IEnumerable).GetEnumerator();
```

Comment:
OH GOSH

Reply:
Let's see if that causes other serialization tests to fail.

Reply:
It didn't locally.